tools/pipelines/templates/include-publish-npm-package-steps.yml

```diff
@@ -88,13 +84,30 @@ steps:
       t1=0
       for f in *.tgz
       do
-          if ! npm publish $f $tag ${{ parameters.publishFlags }}
+          npm publish $f $tag ${{ parameters.publishFlags }} 2>&1 | tee publish_log
+          if grep -q "npm ERR!" "publish_log"
           then
-              let t1+=1
+              if ! grep -q "You cannot publish over the previously published versions" "publish_log"
+              then
+                let t1+=1
```

Comment:
Continue is good for now.  But once we publish topologically, we might want to stop immediately once there is an error.